src/Utilities/InputOutput.f90

```diff
@@ -1221,133 +1199,137 @@ subroutine ParseLine(line, nwords, words, inunit, filename)
-    ! Print 1 layer array with user formatting in wrap format
-    ! **************************************************************************
-    !
-    !    Specifications:
-    ! --------------------------------------------------------------------------
     implicit none
-    ! -- dummy variables
+    ! -- dummy
     integer(I4B), intent(in) :: ncol, nrow, kstp, kper, ilay, iout
-    real(DP),dimension(ncol,nrow), intent(in) :: buf
+    real(DP), dimension(ncol, nrow), intent(in) :: buf
     character(len=*), intent(in) :: text
     character(len=*), intent(in) :: userfmt
     integer(I4B), intent(in) :: nvalues, nwidth
     character(len=1), intent(in) :: editdesc
-    ! -- local variables
+    ! -- local
     integer(I4B) :: i, j, nspaces
-    ! formats
-    1 format('1',/2X,A,' IN LAYER ',I3,' AT END OF TIME STEP ',I3, &
-          ' IN STRESS PERIOD ',I4/2X,75('-'))
-    2 format('1',/1X,A,' FOR CROSS SECTION AT END OF TIME STEP',I3, &
-          ' IN STRESS PERIOD ',I4/1X,79('-'))
-    ! ------------------------------------------------------------------
-    !
-    if (iout<=0) return
+    ! -- formats
+    character(len=*), parameter :: fmtmsgout1 = &
+      "('1',/2X,A,' IN LAYER ',I3,' AT END OF TIME STEP ',I3, &
+&        ' IN STRESS PERIOD ',I4/2X,75('-'))"
+    character(len=*), parameter :: fmtmsgout2 = &
+      "('1',/1X,A,' FOR CROSS SECTION AT END OF TIME STEP',I3, &
+&        ' IN STRESS PERIOD ',I4/1X,79('-'))"
+    !
+    if (iout <= 0) return
     ! -- Print a header depending on ILAY
     if (ilay > 0) then
-       write(iout,1) trim(text), ilay, kstp, kper
-    else if(ilay < 0) then
-       write(iout,2) trim(text), kstp, kper
+      write (iout, fmtmsgout1) trim(text), ilay, kstp, kper
+    else if (ilay < 0) then
+      write (iout, fmtmsgout2) trim(text), kstp, kper
     end if
     !
     ! -- Print column numbers.
     nspaces = 0
     if (editdesc == 'F') nspaces = 3
-    call ucolno(1, ncol, nspaces, nvalues, nwidth+1, iout)
+    call ucolno(1, ncol, nspaces, nvalues, nwidth + 1, iout)
     !
     ! -- Loop through the rows, printing each one in its entirety.
-    do i=1,nrow
-      write(iout,userfmt) i,(buf(j,i),j=1,ncol)
+    do i = 1, nrow
+      write (iout, userfmt) i, (buf(j, i), j=1, ncol)
     end do
     !
     ! -- flush file
-    flush(IOUT)
+    flush (IOUT)
     !
-    ! -- return
+    ! -- Return
     return
   end subroutine ulaprufw
 
-  function read_line(iu, eof) result (astring)
-    ! This function reads a line of arbitrary length and returns
-    ! it.  The returned string can be stored in a deferred-length
-    ! character variable, for example:
-    !
-    !    integer(I4B) :: iu
-    !    character(len=:), allocatable :: my_string
-    !    logical :: eof
-    !    iu = 8
-    !    open(iu,file='my_file')
-    !    my_string = read_line(iu, eof)
+  !> @breif This function reads a line of arbitrary length and returns it.
```

Comment:
```suggestion
  !> @brief This function reads a line of arbitrary length and returns it.
```

Reply:
This fix included with f13f213 which also includes [a change to the file suggested](https://github.com/MODFLOW-USGS/modflow6/pull/1552#issuecomment-1898527486) by @mjreno